From 265eaf20510ea736ee36d745cc85c61e6abfe990 Mon Sep 17 00:00:00 2001
From: Ricardo Cerqueira <ricardo@cyngn.com>
Date: Fri, 6 Nov 2015 01:35:34 +0000
Subject: [PATCH 01/20] sepolicy: Let keystore load firmware

 * Partial pick from commit 3aaeeceb270dc6c8af8dd9a95fa8b9e33097ff50

Change-Id: I46bcb9a8be3ad7fccca8128df1a8f1ce7207ead6
---
 legacy-common/keystore.te | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 legacy-common/keystore.te

diff --git a/legacy-common/keystore.te b/legacy-common/keystore.te
new file mode 100644
index 0000000..4f78ad3
--- /dev/null
+++ b/legacy-common/keystore.te
@@ -0,0 +1,2 @@
+# Allow keystore to search and get keymaste.mdt
+r_dir_file(keystore, firmware_file)
-- 
2.7.4

